### ISSUE TITLE
Is this package actively maintained?

### ISSUE THREAD

**Issue by emilianocalzada** [AfterShip/email-verifier#132]
Hello!

what is the current state of this package, is it working well? 

Looking forward to use it for a new saas, currently using “check-if-email-exists” but I would like to use this package since the license is MIT

**Comment by lryong** [comment]
Yes, the package will be actively maintained and there will be no changes to the license for the time being, but expect the community to help maintain it as well

**Comment by emilianocalzada** [comment]
@lryong 

Thank you for maintaining this repository. I'm building my SaaS application using it and will for sure contribute 

I noticed there are separate verification methods for Yahoo and Gmail APIs. I'm wondering:

1. Why are these implemented as separate methods?
2. Would standard SMTP verification work for these providers as well?

Looking forward to your clarification

**Comment by NexaVerse-AI** [comment]
Some providers have special requirements and they need to be satisfied
that’s why it’s handled separately

**Comment by lryong** [comment]
@emilianocalzada 
Yes, as @NexaVerse-AI  said just now, some service providers have some ways to check, so that is why the community is needed to help build this library. The second problem is yes, but there may be problems in the use process need to be solved by the users themselves
